File: 1005.cs
using System;
class URI {
    static void Main(string[] args) { 
        double nota1 = double.Parse(Console.ReadLine()); double nota2 = double.Parse(Console.ReadLine());
        double media = ((nota1 * 3.5) + (nota2 * 7.5)) / 11;
        Console.WriteLine($"MEDIA = {media:F5}");
    }

}

File: 1005.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    double nota1, nota2, media;
    cin >> nota1;
    cin >> nota2;
    media = ((nota1 * 3.5) + (nota2 * 7.5)) / 11;
    cout << fixed << setprecision(5);
    cout << "MEDIA = " << media << endl;
}
